packages/nimble-components/src/table-column/number-text/types.ts

```diff
@@ -7,3 +7,15 @@ export const NumberTextFormat = {
 } as const;
```

Comment:
This is out-of-scope for this PR, but it seems weird to me that `NumberTextFormat` has a `roundToInteger` value. I would think that would be orthogonal to other formatting options (and so would be its own separate boolean attribute). Do we have a sense for what other values `NumberTextFormat` might have in the future?

Reply:
The currently approved API for the column can be found [here](https://github.com/ni/nimble/blob/886850629165bf624f46fb8a1f43c9ed0cb65133/packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md#api). However, the current direction we're leaning is to remove the `custom` format option since we've found that it is challenging (and in some cases impossible) to use `Intl.NumberFormat` alone to format a number the way we want.

The only other format in the spec right now is `decimal`, which adds a `decimal-digits` field to specify the number of decimals to show. However, `roundToInteger` is really the `decimal` format with `decimal-digits` set to `0`, so I've been starting to question the usefulness of having `roundToInteger` once `decimal` is implemented. I might end up proposing that we remove it.

I don't know if that answers your question, but I think if it doesn't, we should probably have a larger in-person discussion with the dev team to talk about some of the items in the approved API since we've started realizing that it doesn't make as much sense as we originally thought.